### PR TITLE
KO : translate site/ko/guide/random_numbers.ipynb

### PR DESCRIPTION
**Pull request by Myeongjoon**
Thank you!

### CODE REVIEW

File: site/ko/guide/random_numbers.ipynb [hunk 1]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "zSAp2BMj1JZ6"
+      },
+      "source": [
+        "명백한 상태에서 시작하는 제너레이터를 생성하는 방법도 있으나 해당 가이드에서는 다루지 않습니다.\n",
+        "\n",
+        "전역 제너레이터를 사용하기 위해서 `tf.random.get_global_generator` 를 사용할 경우, 디바이스 환경에 유의하셔야 합니다. 전역 제너레이터는 `tf.random.get_global_generator`가 처음 호출될때 (비 결정 상태로) 생성 되고 기본 디바이스에 배치 됩니다. 예를 들어서 `tf.random.get_global_generator`를 `tf.device(\"gpu\")` 영역에서 처음으로 호출하였을 경우, 전역 제너레이ㅌ는 GPU에 할당이 되며, 추후에 CPU에서 사용할시에 GPU-CPU간 복제를 하게 됩니다.\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "전역 제너레이터를 사용하기 위해서 `tf.random.get_global_generator`를 사용할 경우, 디바이스 환경에 유의해야 합니다. 전역 제너레이터는 `tf.random.get_global_generator`가 처음 호출될때 (비 결정 상태로) 생성 되고 기본 디바이스에 배치 됩니다. 예를 들어서 `tf.random.get_global_generator`를 `tf.device(\"gpu\")` 영역에서 처음으로 호출하였을 경우, 전역 제너레이터는 GPU에 할당 되며, 추후에 CPU에서 사용할시에 GPU-CPU간 복제를 하게 됩니다.\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 2]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "제너레이터를 다른 객체로 변경하기 위한 `tf.random.set_global_generator` 함수도 있습니다. 이 함수는 조심히 사용하셔야 합니다. `tf.function`가 이전의 제너레이터를 사용하고 있을 수 있으며 (약한 참조), 이를 변경하는 것은 가비지 콜렉션을 발생 시킬 수 있고, `tf.function`에 문제가 생길 수 있습니다. 전역 제너레이터를 재설정 하는데에 더 좋은 방법은 `Generator.reset_from_seed` 와 같이 새로운 제너레이터를 생성하지 않는 \"reset\" 함수를 사용하는 것 입니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "제너레이터를 다른 객체로 변경하기 위한 `tf.random.set_global_generator` 함수도 있습니다. 이 함수는 조심히 사용해야합니다. `tf.function`가 이전의 제너레이터를 사용하고 있을 수 있으며 (약한 참조), 이를 변경하는 것은 가비지 콜렉션(garbage collection)을 발생시켜 `tf.function`에 문제를 유발할 수 있습니다. 전역 제너레이터를 재설정 하는데에 더 좋은 방법은 `Generator.reset_from_seed` 와 같이 새로운 제너레이터를 생성하지 않는 \"reset\" 함수를 사용하는 것 입니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "dqOaGVzKOsRJ"
+      },
+      "source": [
+        "`split`은 RNG의 `normal`과 같이 제너레이터의 (위의 예제에서는 `g`) 상태를 변경 합니다. 서로 독립인것과 더불어 새로운 제너레이터는 (`new_gs`) 또한 이전 제너레이터와 독립임을 보장 합니다(`g`).\n",
+        "\n",
+        "새로운 제너레이터를 생성 하는 것은 디바이스간 복제의 오버헤드를 피하기 위해 사용하고 있는 제너레이터가 서로 다른 연산에서 동일한 디바이스에 있음을 확실히 하고 싶을때 유용하다. 예를 들어: "
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "새로운 제너레이터를 생성 하는 것은 디바이스간 복제의 오버헤드를 피하기 위해 사용하고 있는 제너레이터가 서로 다른 연산에서 동일한 디바이스에 있음을 확실히 하고 싶을때 유용합니다. 예를 들어: "
```

File: site/ko/guide/random_numbers.ipynb [hunk 4]
```diff
@@ -0,0 +1,669 @@
...
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "5jSnJBlUQzF3"
+      },
+      "outputs": [],
+      "source": [
+        "with tf.device(\"cpu\"):  # \"cpu\"를 사용하고 싶은 디바이스로 변경\n",
+        "  g = tf.random.get_global_generator().split(1)[0]  \n",
+        "  print(g.normal([]))  # 전역 제너레이터와는 다르게 g를 사용하는 것은 디바이스간 복제를 발생하지 않음"
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "  print(g.normal([]))  # 전역 제너레이터와는 다르게 g를 사용하는 것은 디바이스간 복제를 발생하지 않습니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 5]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "sCxbccYMRdd4"
+      },
+      "source": [
+        "Note: 이론적으로, `split` 대신에 `from_seed` 생성자(Constructor)를 사용할 수 있습니다. 그러나 이는 새로운 제너레이터가 전역 제너레이터에 독립임을 보장하지 않습니다. 또한 두 제너레이터의 시드가 동일 하거나 랜덤 생성 스트림이 겹치는 시드를 생성하는 등의 위험이 있습니다.\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "참고: 이론적으로, `split` 대신에 `from_seed` 생성자(Constructor)를 사용할 수 있습니다. 그러나 이는 새로운 제너레이터가 전역 제너레이터에 독립임을 보장하지 않습니다. 또한 두 제너레이터의 시드가 동일 하거나 랜덤 생성 스트림이 겹치는 시드를 생성하는 등의 위험이 있습니다.\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 6]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "`split`을 split된 제너레이터를 통해서 호출하면 재귀적으로 split 할 수 있습니다. 재귀 깊이의 제한은 (오버 플로우는 방지) 없습니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "`split`을 분할(split)된 제너레이터를 통해서 호출하면 재귀적으로 분할할 수 있습니다. 재귀 깊이의 제한은 없지만, 오버플로우는 방지하도록 되어있습니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 7]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "jnSjhY6WM-J8"
+      },
+      "source": [
+        "#### `tf.function` 밖에서 제너레이터를 생성 \n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "#### `tf.function` 밖에서 제너레이터 생성하기 \n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 8]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "PwIrBv_zUYwI"
+      },
+      "source": [
+        "#### `tf.function` 안에서 제너레이터 생성\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "#### `tf.function` 안에서 제너레이터 생성하기\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 9]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "`tf.function` 안에서 제너레이터를 생성하는 경우는 오직 함수의 첫번째 실행에서만 실행 됩니다. "
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "`tf.function` 안에서 제너레이터를 생성하는 경우는 오직 함수의 첫번째 호출에서만 실행 됩니다. "
```

File: site/ko/guide/random_numbers.ipynb [hunk 10]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "UaTVnOhHVM9a"
+      },
+      "source": [
+        "#### 제너레이터를 `tf.function`의 파라미터로 보내기\n",
+        "\n",
+        "`tf.function`의 파라미터로 사용될 경우, 동일한 상태 사이즈를 (상태 사이즈는 RNG 알고리즘에 의해 결정됨) 가진 서로 다른 제너레이터 객체는 `tf.function`를 재추적(retracing) 하지 않습니다, 다른 상태 사이즈를 가질 경우에는 동작 합니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "`tf.function`의 파라미터로 사용될 경우, 동일한 상태 사이즈를 (상태 사이즈는 RNG 알고리즘에 의해 결정됩니다) 가진 서로 다른 제너레이터 객체는 `tf.function`를 재추적(retracing) 하지 않습니다. 반면, 다른 상태 사이즈를 가질 경우에는 동작 합니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 11]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GyZv9QJkZfkQ"
+      },
+      "source": [
+        "#### 분산 전략 밖에서 제너레이터 생성\n",
+        "\n",
+        "제너레이터가 전략 스콥 밖에서 생성될 경우, 제너레이터에 대한 모든 복제(replicas)의 접근이 직렬화(serialized) 되고 따라서 복제들은 서로 다른 랜점 숫자를 가지게 됩니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "제너레이터가 전략 스코프(scope) 밖에서 생성될 경우, 제너레이터에 대한 모든 복제(replicas)의 접근이 직렬화(serialized) 되고 따라서 복제들은 서로 다른 랜점 숫자를 가지게 됩니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 12]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Yal4LbBKbAeN"
+      },
+      "source": [
+        "#### 분산 전략(distribution strategies)안에서 제너레이터 생성\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "#### 분산 전략(distribution strategies)안에서 제너레이터 생성하기\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 13]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "전략 영역(strategy scopes)안에서 제너레이터를 생성하는 것은 허용되지 않았습니다. 이는 제너레이터를 어떻게 복제하는지에 대한 모호함이 있기 때문입니다. (예를 들어 각 복제본이 동일한 랜점 숫자를 가지게 복제를 하거나 서로 다른 랜덤 숫자를 가지도록 'split' 복제를 하는 것)."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "전략 영역(strategy scopes)안에서 제너레이터를 생성하는 것은 허용되지 않았습니다. 이는 제너레이터를 어떻게 복제하는지에 대한 모호함이 있기 때문입니다 (예를 들어 각 복제본이 동일한 랜점 숫자를 가지게 복제를 하거나 서로 다른 랜덤 숫자를 가지도록 `split` 복제를 하는 것에 대한 모호함이 있습니다)."
```

File: site/ko/guide/random_numbers.ipynb [hunk 14]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ODLS8njzbUEF"
+      },
+      "source": [
+        "#### 제너레이터를 `Strategy.run`의 파라미터로 사용\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "#### 제너레이터를 `Strategy.run`의 파라미터로 사용하기\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 15]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "각 복제본이 각자의 제너레이터를 사용하길 원할 경우, `n`개의 제너레이터가 필요 합니다. (각각 복제하거나 split). `n`은 복제의 갯수이며 , 이를 `Strategy.run`의 파라미터로 보냅니다.\n"
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "각 복제본이 각자의 제너레이터를 사용하길 원할 경우, `n`개의 제너레이터가 필요 합니다 (각각 복제하거나 split). `n`은 복제의 갯수이며 , 이를 `Strategy.run`의 파라미터로 보냅니다.\n"
```

File: site/ko/guide/random_numbers.ipynb [hunk 16]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "73an1POpsi6V"
+      },
+      "source": [
+        "## 무상태 RNGs\n",
+        "\n",
+        "무상태 RNGs의 사용법은 간단합니다. 순수 함수이기 때문에 상태나 사이드 이펙트가 없습니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "무상태 RNGs의 사용법은 간단합니다. 순수 함수이기 때문에 부작용이 없습니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 17]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "58-8kvR4pRwO"
+      },
+      "source": [
+        "### 일반\n",
+        "\n",
+        "`tf.random.Generator` 클래스와 `stateless` 함수는 모든 디바이스에서 필록스(Philox) 알고리즘을 지원 합니다. (`\"philox\"` 또는 `tf.random.Algorithm.PHILOX`)\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "`tf.random.Generator` 클래스와 `stateless` 함수는 모든 디바이스에서 필록스(Philox) 알고리즘을 지원 합니다 (`\"philox\"` 또는 `tf.random.Algorithm.PHILOX`로 명시할 수 있습니다).\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 18]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BlGY1iiph_C2"
+      },
+      "source": [
+        "텐서플로는 `tf.random` 모듈에서 유사 난수 생성기(pseudo random number generator, RNG)를 제공 합니다. 이 문서에서는 난수 생서기를 다루는 방법과 이 기능이 어떻게 다른 텐서플로의 서브 시스템과 상호작용 하는지 설명합니다. \n",
+        "\n",
+        "텐서플로는 난수 생성 프로세스를 다루기 위한 두 가지 방식을 제공합니다.:\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "텐서플로는 난수 생성 프로세스를 다루기 위한 두 가지 방식을 제공합니다:\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 19]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "1. `tf.random.Generator` 객체 사용을 통한 방식. 각 객체는 상태를 (`tf.Variable` 안에) 유지합니다. 이 상태는 매 숫자 생성때마다 변하게 됩니다.\n",
+        "\n",
+        "2. `tf.random.stateless_uniform`와 같은 순수-함수형 무상태 랜덤 함수를 통한 방식. 같은 디바이스에서 동일한 인수를 (시드값 포함) 통해 해당 함수를 호출 하면 항상 같은 결과를 출력 합니다.\n",
+        "\n",
+        "주의: `tf.random.uniform` 와 `tf.random.normal` 같은 구버전 TF 1.x의 RNG들은 아직 삭제되지 않았지만 사용을 권장하지 않습니다.\n",
+        "\n",
+        "주의: 랜덤 함수는 텐서 플로 버전에 따라 동일함을 보장하지 않습니다. 참조: [Version Compatibility](https://www.tensorflow.org/guide/versions#what_is_not_covered)"
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "주의: 랜덤 함수는 텐서 플로 버전에 따라 동일함을 보장하지 않습니다. 참조: [버전 호환성](https://www.tensorflow.org/guide/versions#what_is_not_covered)"
```

File: site/ko/guide/random_numbers.ipynb [hunk 20]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "QmRCeAvTxulW"
+      },
+      "source": [
+        "객체를 생성하는데에는 여러 방법이 있습니다. 위에서 볼 수 있는것 처럼 가장 쉬운 방법은 `Generator.from_seed`를 사용하는 것 이며 이는 시드를 통해서 제너레이터를 생성 합니다. 시드는 0 이상의 정수형 입니다. `from_seed`는 `alg`를 추가적으로 전달 받을 수 있으며 이는 제너레이터가 사용할 RNG 알고리즘 입니다.:"
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "객체를 생성하는데에는 여러 방법이 있습니다. 위에서 볼 수 있는것 처럼 가장 쉬운 방법은 `Generator.from_seed`를 사용하는 것 이며 이는 시드를 통해서 제너레이터를 생성 합니다. 시드는 0 이상의 정수형 입니다. `from_seed`는 `alg`를 추가적으로 전달 받을 수 있으며 이는 제너레이터가 사용할 RNG 알고리즘 입니다:"
```

File: site/ko/guide/random_numbers.ipynb [hunk 21]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BlGY1iiph_C2"
+      },
+      "source": [
+        "텐서플로는 `tf.random` 모듈에서 유사 난수 생성기(pseudo random number generator, RNG)를 제공 합니다. 이 문서에서는 난수 생서기를 다루는 방법과 이 기능이 어떻게 다른 텐서플로의 서브 시스템과 상호작용 하는지 설명합니다. \n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "텐서플로는 `tf.random` 모듈에서 유사 난수 생성기(pseudo random number generator, RNG)를 제공 합니다. 이 문서에서는 난수 생성기를 다루는 방법과 이 기능이 어떻게 다른 텐서플로의 서브 시스템과 상호작용 하는지 설명합니다. \n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 22]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "eqMlrUsVu2Ai"
+      },
+      "source": [
+        "## `tf.random.Generator` 클래스\n",
+        "\n",
+        "각 RNG 호출마다 다른 결과를 생성하기 원할 경우 `tf.random.Generator` 클래스를 사용할 수 있습니다. 이는 내부 상태를 (`tf.Variable` 객체가 관리) 유지 합니다. 이 상태는 랜덤 숫자가 생성될때마다 업데이트 됩니다. 상태가 `tf.Variable`에 의해 유지되기 때문에, 쉬운 체크포인팅, 자동적인 컨트롤 종속, 쓰레드 안전성의 장점이 있습니다.\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "각 RNG 호출마다 다른 결과를 생성하기 원할 경우 `tf.random.Generator` 클래스를 사용할 수 있습니다. 이는 내부 상태를 (`tf.Variable` 객체가 관리) 유지 합니다. 이 상태는 난수가 생성될때마다 업데이트 됩니다. 상태가 `tf.Variable`에 의해 유지되기 때문에, 쉬운 체크포인팅(checkpointing), 자동적인 컨트롤 종속, 쓰레드(thread) 안전성의 장점이 있습니다.\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 23]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "zSAp2BMj1JZ6"
+      },
+      "source": [
+        "명백한 상태에서 시작하는 제너레이터를 생성하는 방법도 있으나 해당 가이드에서는 다루지 않습니다.\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "명백한 상태(explicit state)에서 제너레이터를 생성하는 방법 등 제너레이터를 초기화하는 방법은 여러가지가 있지만 해당 가이드에서는 다루지 않습니다.\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 24]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "전역 제너레이터를 사용하기 위해서 `tf.random.get_global_generator`를 사용할 경우, 디바이스 환경에 유의해야 합니다. 전역 제너레이터는 `tf.random.get_global_generator`가 처음 호출될때 (비 결정 상태로) 생성 되고 기본 디바이스에 배치 됩니다. 예를 들어서 `tf.random.get_global_generator`를 `tf.device(\"gpu\")` 영역에서 처음으로 호출하였을 경우, 전역 제너레이터는 GPU에 할당 되며, 추후에 CPU에서 사용할시에 GPU-CPU간 복제를 하게 됩니다.\n",
+        "\n",
+        "제너레이터를 다른 객체로 변경하기 위한 `tf.random.set_global_generator` 함수도 있습니다. 이 함수는 조심히 사용해야합니다. `tf.function`가 이전의 제너레이터를 사용하고 있을 수 있으며 (약한 참조), 이를 변경하는 것은 가비지 콜렉션(garbage collection)을 발생시켜 `tf.function`에 문제를 유발할 수 있습니다. 전역 제너레이터를 재설정 하는데에 더 좋은 방법은 `Generator.reset_from_seed` 와 같이 새로운 제너레이터를 생성하지 않는 \"reset\" 함수를 사용하는 것 입니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "제너레이터를 다른 객체로 변경하기 위한 `tf.random.set_global_generator` 함수도 있습니다. 이 함수는 조심히 사용해야합니다. `tf.function`가 이전의 제너레이터를 (약한 참조로) 사용하고 있을 수 있으며, 이를 변경하는 것은 가비지 콜렉션(garbage collection)을 발생시켜 `tf.function`에 문제를 유발할 수 있습니다. 전역 제너레이터를 재설정 하는데에 더 좋은 방법은 `Generator.reset_from_seed` 와 같이 새로운 제너레이터를 생성하지 않는 \"리셋\" 함수를 사용하는 것 입니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 25]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "dqOaGVzKOsRJ"
+      },
+      "source": [
+        "`split`은 RNG의 `normal`과 같이 제너레이터의 (위의 예제에서는 `g`) 상태를 변경 합니다. 서로 독립인것과 더불어 새로운 제너레이터는 (`new_gs`) 또한 이전 제너레이터와 독립임을 보장 합니다(`g`).\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "`split`은 RNG의 `normal`과 같이 제너레이터의 (위의 예제에서는 `g`) 상태를 변경 합니다. 서로 독립인것과 더불어 새로운 제너레이터는 (`new_gs`) 또한 이전 제너레이터와 독립임을 보장 합니다 (`g`).\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 26]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "L_8kC7kbO5uu"
+      },
+      "source": [
+        "사용자는 함수를 호출할때 제너레이터 객체가 여전히 살아 있음을 (가비지 콜렉션이 되지 않음) 확실히 해야 합니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "사용자는 함수를 호출할때 제너레이터 객체가 여전히 살아 있음을 확실히 해야 합니다 (가비지 콜렉션이 되지 않아야 합니다)."
```

File: site/ko/guide/random_numbers.ipynb [hunk 27]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "UaTVnOhHVM9a"
+      },
+      "source": [
+        "#### 제너레이터를 `tf.function`의 파라미터로 보내기\n",
+        "\n",
+        "`tf.function`의 파라미터로 사용될 경우, 동일한 상태 사이즈를 (상태 사이즈는 RNG 알고리즘에 의해 결정됩니다) 가진 서로 다른 제너레이터 객체는 `tf.function`를 재추적(retracing) 하지 않습니다. 반면, 다른 상태 사이즈를 가질 경우에는 동작 합니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "`tf.function`의 파라미터로 사용될 경우, 동일한 상태 사이즈를 가진 서로 다른 제너레이터 객체는 `tf.function`를 재추적(retracing) 하지 않습니다 (상태 사이즈는 RNG 알고리즘에 의해 결정됩니다). 반면, 다른 상태 사이즈를 가질 경우에는 동작 합니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 28]
```diff
@@ -0,0 +1,669 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "z9H0wuvp9VwH"
+      },
+      "source": [
+        "### 독립적인 랜덤 숫자 스트림 생성\n",
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "### 독립적인 난수 스트림 생성\n",
```

File: site/ko/guide/random_numbers.ipynb [hunk 29]
```diff
@@ -0,0 +1,669 @@
...
+        "\n",
+        "많은 어플리케이션에서는 서로 겹치지 않으며 통계적으로 상관관계를 가지지 않는 여러개의 독립적인 랜덤 숫자 스트림이 필요 합니다. 이는 각각 독립이 보장된 여러 제너레이터를 생성 하는 `Generator.split`를 통해서 해결 할 수 있습니다. (예를 들어 독립 스트림 생성)."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "많은 어플리케이션에서는 서로 겹치지 않으며 통계적으로 상관관계를 가지지 않는 여러개의 독립적인 난수 스트림이 필요 합니다. 이는 각각 독립이 보장된 여러 제너레이터를 생성 하는 `Generator.split`를 통해서 해결 할 수 있습니다 (즉 독립 스트림 생성할 수 있습니다)."
```

File: site/ko/guide/random_numbers.ipynb [hunk 30]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "73an1POpsi6V"
+      },
+      "source": [
+        "## 무상태 RNGs\n",
+        "\n",
+        "무상태 RNGs의 사용법은 간단합니다. 순수 함수이기 때문에 부작용이 없습니다."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "무상태(stateless) RNGs의 사용법은 간단합니다. 순수 함수이기 때문에 부작용이 없습니다."
```

File: site/ko/guide/random_numbers.ipynb [hunk 31]
```diff
@@ -0,0 +1,669 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Yal4LbBKbAeN"
+      },
+      "source": [
+        "#### 분산 전략(distribution strategies)안에서 제너레이터 생성하기\n",
+        "\n",
+        "전략 영역(strategy scopes)안에서 제너레이터를 생성하는 것은 허용되지 않았습니다. 이는 제너레이터를 어떻게 복제하는지에 대한 모호함이 있기 때문입니다 (예를 들어 각 복제본이 동일한 랜점 숫자를 가지게 복제를 하거나 서로 다른 랜덤 숫자를 가지도록 `split` 복제를 하는 것에 대한 모호함이 있습니다)."
```
Comment (jaketae) [inline site/ko/guide/random_numbers.ipynb]:
```suggestion
        "전략 영역(strategy scopes)안에서 제너레이터를 생성하는 것은 허용되지 않았습니다. 이는 제너레이터를 어떻게 복제하는지에 대한 모호함이 있기 때문입니다 (예를 들어 각 복제본이 동일한 난수를 갖도록 복제를 하거나 서로 다른 난수를 갖도록 `split` 복제를 하는지에 대한 모호함이 있습니다)."
```

**Comment by jaketae** [comment]
Thank you for the fix. LGTM

File: site/ko/guide/random_numbers.ipynb [hunk 32]
```diff
@@ -0,0 +1,669 @@
...
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs-l10n/blob/master/site/ko/guide/random_numbers.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃허브(GitHub)에서 소스 보기</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs-l10n/site/ko/guide/random_numbers.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />노트북 다운로드하기</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
```
Comment (lamberta) [inline site/ko/guide/random_numbers.ipynb]: Thanks. Can you please add the community translation disclaimer, as seen here: https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/ko/tutorials/quickstart/beginner.ipynb#scrollTo=_dt4BrWOPl-C

**Comment by jaketae** [comment]
I shouldn’t have missed that. Thanks for pointing it out @lamberta

**Comment by lamberta** [comment]
No worries 😀 Maybe we can look into adding some GitHub CI checks for common patterns

**Comment by Myeongjoon** [comment]
@jaketae, @lamberta 
Thank you. I added community translation disclaimer

**Pull request merged**